src/app/features/tech-record/components/tech-record-summary-changes/tech-record-summary-changes.component.ts

```diff
@@ -96,6 +96,10 @@ export class TechRecordSummaryChangesComponent implements OnInit, OnDestroy {
       .pipe(take(1), takeUntil(this.destroy$))
       .subscribe((changes) => {
         this.techRecordChanges = changes;
+        if (this.vehicleType === VehicleTypes.PSV || this.vehicleType === VehicleTypes.HGV) {
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          delete (this.techRecordChanges as any).techRecord_numberOfWheelsDriven;
+        }
```

Comment:
I know this looks weird but numberOfWheelsDriven is in the template for HGV and PSV (even though they're hidden) it ends up appearing as an empty field in vehicle summary. it seems to be something to do with tests and I don't want to break them for the sake of this bug.